android/index.php

```diff
@@ -12,7 +12,8 @@
     'showMenu' => true,
     'banner' => [
       'title' => 'Type in Your Language On Your Android Device!',
-      'button' => '<a href="'.$playstore.'"><img id="app-store" src="https://developer.android.com/images/brand/en_app_rgb_wo_60.png" alt="Android app on Google Play"></a>',
+      'button' => '<a href="'.$playstore.'"><img id="app-store" src="cdn(en_app_rgb_wo_60.png)"
```

Comment:
This doesn't look right to me? But I see that it is, wow, that's pretty magical and horrible:

https://github.com/keymanapp/keyman.com/blob/a2c22da27126d69a23f8fa7b66b0f94bd75140b0/_includes/includes/template.php#L127-L130

Definite candidate for rewrite in the future!

Reply:
I found that piece of magic from 
https://github.com/keymanapp/keyman.com/blob/9ca42f7f62ffd31ed2e9038f4d0ab992289e2e25/iphone-and-ipad/index.php#L15

Reply:
Yeah, I saw that too, and then had to dig because I couldn't quite believe it... not sure who wrote it now, and doesn't really matter, but it's certainly an unnecessarily magical way to solve the problem!